lib/algolia.ex

```diff
@@ -168,11 +167,19 @@ defmodule Algolia do
           path_encode_fun: &URI.encode/1,
           connect_timeout: 3_000 * (curr_retry + 1),
           recv_timeout: 30_000 * (curr_retry + 1),
-          ssl_options: [{:versions, [:"tlsv1.2"]}]
+          ssl_options: [
+            {:versions, [:"tlsv1.2"]},
+            {:cacerts, :public_key.cacerts_get()},
+            {:customize_hostname_check,
+             [
+               match_fun: :public_key.pkix_verify_hostname_match_fun(:https)
+             ]}
+          ]
```

Comment:
Do we need to bump the minimum version of any of our dependencies to support these new options?

Also, could this be a keyword list?

```elixir
[
  versions: [:"tlsv1.2"],
  cacerts: :public_key.cacerts_get()},
  customize_hostname_check:
             [
               match_fun: :public_key.pkix_verify_hostname_match_fun(:https)
             ]
]
```

Reply:
The list of tuples is fine. I'm just thinking the formatter would most likely change it to a keyword list syntax.